src/mobility/components/OperatorNameAndLogo.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {Image, View} from 'react-native';
+import {ThemeText} from '@atb/components/text';
+import {useRemoteConfig} from '@atb/RemoteConfigContext';
+import {SvgCssUri} from 'react-native-svg';
+import {StyleSheet} from '@atb/theme';
+
+/**
+ * Large logo is shown at the empty part of the bottom sheet.
+ * Small logo is shown near the operator name.
+ * 
+ * Please see figma page below for reference
+ * https://www.figma.com/file/zdZwvobgpEWSagKt0tderx/App?node-id=20467-42962
+ */
+const LOGO_SIZES = {"small": 20, "large": 50}
+
+/**
+ * @param operatorName will only be displayed on small logo
+ * 
+ * @param logoUrl logo to be displayed, can be SVG or PNG, 
+ *                if SVG, show using <SvgCssUri>, otherwise use <Image> 
+ * 
+ * @param logoSize determines small or large logo (see Figma)
+ * 
+ * @param fallback logo to be used when the logo URL is null
+ * 
+ */
+type OperatorNameAndLogoProps = {
+  operatorName: string;
+  logoUrl: string | undefined;
+  logoSize?: "small" | "large";
+  fallback?: JSX.Element
+};
+
+export const OperatorNameAndLogo = ({
+  operatorName,
+  logoUrl,
+  logoSize = "small",
+  fallback
+}: OperatorNameAndLogoProps) => {
```

Comment:
I don't think this should be a single component, since it has such a wide variety of how it can end up rendered (can show nothing, just name, small image and name, large image and no name).

Reply:
Split into BrandingImage.tsx